docs/release-notes/self-hosted/self-hosted-v5.0.0.md

```diff
@@ -0,0 +1,96 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v5.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.9.16
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.28
+---
+
+# Self-hosted v5.0.0
+
+These release notes are for [Codacy Self-hosted v5.0.0](https://github.com/codacy/chart/releases/tag/5.0.0){: target="_blank"}, released on December 14, 2021. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Breaking changes
```

Comment:
We also have another breaking change: hotspots-api, hotspots-worker and codacy-activities were dropped, with this there is no longer a requirement for the hotspots DB and the codacy-activities DB.

Reply:
Thanks, @pedrobpereira. Do we need to ask the customers to perform any additional steps while updating to 5.0.0 or is the update process transparent for them?

Reply:
we expect (it is what happened in our tests) that the deployments in the cluster will be deleted with the upgrade.

Everything should be transparent, but there are things to cleanup:
- dbs for activities and hotspots
- properties that have been removed from the values files that were configuring these components

Nothing is breaking though

Reply:
@pedrobpereira and @pedrocodacy I pushed a new commit mentioning these changes and recommending that customers manually remove the databases and clean up the configuration files.

I based the list of configuration file keys that should be cleaned up on https://github.com/codacy/chart/pull/705, but please validate.